shrine_with_cart/mdc_100_series/lib/home.dart

```diff
@@ -18,18 +18,41 @@ import 'package:scoped_model/scoped_model.dart';
 import 'model/app_state_model.dart';
 import 'model/product.dart';
 import 'supplemental/asymmetric_view.dart';
+import 'backdrop.dart';
+import 'short_bottom_sheet.dart';
 
-class HomePage extends StatelessWidget {
+class ProductPage extends StatelessWidget {
```

Comment:
I feel like it would be nicer to have users refactor the HomePage class to this, but the name could propagate instead if that makes more sense. I'm not super fond of the name `ProductPage`, just needed something so that `HomePage` could become the `Stack`.

Reply:
Then let's leave it as is and have them change.